### PR TITLE
Support argo simple recipes

### LINKED ISSUE #25: ogdc-runner: add support for rendering "simple" recipe into argo workflow
Add support for "simple" recipes. This would let the runner take a "simple" recipe (e.g., see https://github.com/QGreenland-Net/ogdc-recipes/tree/411e41f2f7e6310044cfc26adafb741eb77982e1/recipes/sea-ice-conc) and convert it into an argo workflow that can be submitted to the cluster.

* [x] Add support to render a "simple" recipe into an argo workflow and submit it to the cluster
* [x] Add unit test(s) for validation and rendering of "simple" recipe into argo workflow
* [ ] Add an integration test that asserts successful workflow execution given a "simple" recipe.

### PR DESCRIPTION
**Pull request by trey-stafford**
* Remove OBE `parsl` code/config
* Add code to run ogdc "simple" recipes with argo via [hera](https://hera.readthedocs.io/).
* Add documentation on how to run the CLI to submit a "simple" recipe

Resolves #25 and #24 

See also: https://github.com/QGreenland-Net/ogdc-recipes/pull/4

Some TODOs that have come up while developing this:

* https://github.com/QGreenland-Net/ogdc-runner/issues/31
* https://github.com/QGreenland-Net/ogdc-runner/issues/32

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Docker image for running gdal/ogr and fetch (wget) commands for ogdc recipes.
+FROM ghcr.io/osgeo/gdal:alpine-normal-latest
+# We use wget to fetch data from remote sources.
+RUN apk update && apk add wget
```
Comment (trey-stafford) [inline Dockerfile:4]: The only reason we need a custom docker image for the simple examples I've been testing against is the need to fetch data with `wget`. Most docker images don't come with it, and some (like `alpine`, as in this case) ship with the busybox version. This lacks some options we need for QGreenland data layers (`--content-disposition`), including the [seal-tags](https://github.com/QGreenland-Net/ogdc-recipes/blob/e4c5e3aebc83ccc68da9170106dce9ecc9634c54/recipes/seal-tags/meta.yml#L9) example.

Perhaps there's a different/better way to approach fetching HTTP assets. I considered using `hera`'s `HTTPArtifact` as input, but this also lacks the option of fetching a remote asset into a specified directory without giving the fetched thing a name. 

Longer term, I imagine we'll want other options as well (and maybe even want to use `hera`'s [RunnerScriptConstructor](https://hera.readthedocs.io/en/stable/api/workflows/hera/?h=runnerscript#hera.workflows.RunnerScriptConstructor) (see also: https://hera.readthedocs.io/en/stable/user-guides/script-basics/) ). So this image may very well grow more complex.

File: .github/CONTRIBUTING.md
Comment (rmarow) [inline .github/CONTRIBUTING.md:1]: lets move this to docs and reference it in the Readme

**Review by rmarow — APPROVED** [review]

File: .github/CONTRIBUTING.md
Comment (rmarow) [reply .github/CONTRIBUTING.md:1]: note: we also want to reference https://github.com/QGreenland-Net/ogdc-argo for instillation

**Review by rmarow — APPROVED** [review]

File: .github/CONTRIBUTING.md
Comment (trey-stafford) [reply .github/CONTRIBUTING.md:1]: > lets move this to docs and reference it in the Readme

62db83a

Comment (trey-stafford) [reply .github/CONTRIBUTING.md:1]: > note: we also want to reference https://github.com/QGreenland-Net/ogdc-argo for instillation

bf633a5

**Pull request merged**